users/bifbofii/keyrecords/keycodes.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2022 Christoph Jabs (BifbofII)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "bifbofii.h"
+
+enum userspace_custom_keycodes {
+    KC_QWERTY                   = SAFE_RANGE, // Sets default layer to QWERTY
```

Comment:
Due to the recent changes for keycodes, I'd recommend this over safe range, since safe range starts in the kb block, and this starts in the user block.
```suggestion
    KC_QWERTY                   = QK_USER, // Sets default layer to QWERTY
```